pkg/operator/starter.go

```diff
@@ -48,8 +50,13 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 	if err != nil {
 		return err
 	}
+	opClient, err := opv1client.NewForConfig(cc.KubeConfig)
+	if err != nil {
+		return err
+	}
 
 	configInformers := configinformers.NewSharedInformerFactory(configClient, 10*time.Minute)
+	opInformers := opinformers.NewSharedInformerFactory(opClient, 10*time.Minute)
 	kubeInformersForNamespaces := v1helpers.NewKubeInformersForNamespaces(kubeClient,
 		"",
 		operatorclient.GlobalUserSpecifiedConfigNamespace,
```

Comment:
I left these variable names alone, trying to avoid refactoring code that this PR does not already touch to keep this 4.13-only patch small and isolated.